iac-extensions/cloudformation/src/test/resources/checks/UnencryptedSqsQueueCheck/UnencryptedSqsQueueCheck.yaml

```
@@ -0,0 +1,36 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Resources:
+  UnencryptedQueue:
+    Type: AWS::SQS::Queue
+    Properties:
+     "DisplayName": "unencrypted_queue"
```

```suggestion
     DisplayName: "unencrypted_queue"
```
Nitpick, unless it was done on purpose?

---

Well, it was like that before, but YAML shouldn't pay attention to quotes on a key.